lib/google.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+create_image_from_snapshot(){
+    IMAGE=$1
```

Comment:
Can we use `local` since those are in a function and you are using bash already?

Reply:
sure, i'll do the change